AndroidAnnotations/androidannotations/src/main/java/org/androidannotations/holder/HasPreferenceHeaders.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (C) 2010-2015 eBusiness Information, Excilys Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed To in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.androidannotations.holder;
+
+import com.sun.codemodel.JBlock;
+import com.sun.codemodel.JVar;
+
+public interface HasPreferenceHeaders extends HasPreferences {
+
```

Comment:
@yDelouis since headers are only used in `PreferenceActivity`, i decided to create a subinterface, so `EFragmentHolder` does not have to implement these methods with empty method blocks.